Doc/library/sqlite3.rst

```diff
@@ -94,6 +94,12 @@ using :meth:`~Cursor.executemany`::
    ... ]
    >>> cur.executemany('INSERT INTO stocks VALUES(?, ?, ?, ?, ?)', data)
 
+Notice that we used question mark placeholders to bind *data* to the query.
```

Comment:
```suggestion
Notice that we used ``?`` placeholders to bind *data* to the query.
```

May as well use the literal here, no? Its shorter, more explicit, and avoids any cross-language interpretation issues.

Reply:
Absolutely!